### DISCUSSION TITLE
RC compatible with 8.7.0-pre

### DISCUSSION THREAD

**Discussion by mirceahasegan** [CardanoSolutions/ogmios#354] [Q & A] [answered]
Hello, and thank you for all your work!
When can we expect a new release candidate compatible with [Node 8.7.0-pre](https://github.com/input-output-hk/cardano-node/releases/tag/8.7.0-pre)?
Thank you

**Comment by KtorZ** [comment, marked as answer]
🤷

**Reply by KtorZ** [reply]
Thanks for the notice though, I somehow missed that new release.
